typing/jkind.ml

```diff
@@ -1706,10 +1720,9 @@ module Jkind_desc = struct
     let ( ({ layout = lay1; mod_bounds = bounds1; with_bounds = No_with_bounds } :
             Allowance.right_only jkind_desc),
           _ ) =
-      Layout_and_axes.normalize ~mode:Ignore_best ~jkind_of_type
-        ~map_type_info:(fun _ ti ->
-          { relevant_axes = Axis_set.diff ti.relevant_axes axes_max_on_right })
-        sub
+      Layout_and_axes.normalize
+        ~relevant_axes:(Axis_set.complement axes_max_on_right)
```

Comment:
as a minor optimization, maybe we switch the argument to `~irrelevant_axes`, to avoid calling `complement` both here and in `normalize`?

or maybe that's too small of an optimization to be worth it